requirements.txt

```diff
@@ -16,6 +16,7 @@ pytest-cov
 nbconvert
 
 # This project pip deps
+jupytext
```

Comment:
i _think_ this is okay. i have `jupytext` included here, but i have removed it from `requirements-package.txt` so that it doesn't get installed for users.

if we remove it from here though, the unit tests will fail. let me know if this isn't the appropriate place for this!

Reply:
Yes, you got it right, this is the right place.